.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: CI
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  test:
+    name: Molecule
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        agent_type:
+          - monitoring
+          - logging
+          - ops-agent
+        scenario:
+          - install_agent
+          - uninstall_agent
+        version:
+          - latest
+          - 6.*.*
+          - 6.1.0
+          - 1.*.*
+          - 1.7.0
+          - 1.1.0
+        distro:
+          - geerlingguy/docker-centos7-ansible
+          - geerlingguy/docker-centos8-ansible
+          # RedHat Universal Base Image (rhel8)
+          - geerlingguy/docker-ubi8-ansible
+          - geerlingguy/docker-amazonlinux2-ansible
+          - geerlingguy/docker-debian9-ansible
+          - geerlingguy/docker-debian10-ansible
+          - geerlingguy/docker-ubuntu1604-ansible
+          - geerlingguy/docker-ubuntu1804-ansible
+          - geerlingguy/docker-ubuntu2004-ansible
+
+        exclude:
+          - scenario: uninstall_agent
```

Comment:
Are these excluded to save some testing resources (aka the uninstall only needs to test once with `latest`)?

Reply:
I assumed if uninstall works with latest it will work with the others. I guess it can't hurt to test a variety of uninstalls though.